api/net/Areas/Editor/Controllers/LookupController.cs

```diff
@@ -111,16 +112,17 @@ public class LookupController : ControllerBase
     [SwaggerOperation(Tags = new[] { "Lookup" })]
     [ETagCacheTableFilter("lookups")]
     [ResponseCache(Duration = 5 * 60)]
-    public IActionResult FindAll()
+    public async Task<IActionResult> FindAllAsync()
     {
+        if (!_keycloakOptions.ClientId.HasValue) throw new ConfigurationException("Keycloak clientId has not been configured");
+
         var actions = _actionService.FindAll();
         var categories = _categoryService.FindAll();
-        var claims = _claimService.FindAll();
         var products = _productService.FindAll();
         var sources = _sourceService.FindAll();
         var license = _licenseService.FindAll();
         var ingestTypes = _ingestTypeService.FindAll();
-        var roles = _roleService.FindAll();
+        var roles = (await _keycloakService.GetRolesAsync(_keycloakOptions.ClientId.Value)).Select(r => r.Name!);
```

Comment:
We fetch the roles from keycloak instead of our database.  This simplifies managing and syncing these objects.